fog/view/server/src/fog_view_router_server.rs

```diff
@@ -74,16 +89,48 @@ where
             mc_util_grpc::HealthService::new(Some(readiness_indicator.into()), logger.clone())
                 .into_service();
 
-        // Package service into grpc server
-        log::info!(
-            logger,
-            "Starting Fog View Router server on {}",
-            config.client_listen_uri.addr(),
-        );
-        let router_server_builder = grpcio::ServerBuilder::new(env.clone())
-            .register_service(fog_view_router_service)
-            .register_service(health_service)
-            .bind_using_uri(&config.client_listen_uri, logger.clone());
+        let router_server_builder = match config.client_listen_uri {
+            RouterClientListenUri::Streaming(ref streaming_uri) => {
+                let fog_view_router_service =
+                    view_grpc::create_fog_view_router_api(FogViewRouterService::new(
+                        enclave.clone(),
+                        shards,
+                        config.chain_id.clone(),
+                        client_authenticator,
+                        logger.clone(),
+                    ));
+                log::debug!(logger, "Constructed Fog View Router streaming GRPC Service");
+                log::info!(
+                    logger,
+                    "Starting Fog View Router unary server on {}",
```

Comment:
:wrench: Should this be the streaming server?
```suggestion
                    "Starting Fog View Router streaming server on {}",
```

Reply:
Done.